docker/Dockerfile.nvidia

```diff
@@ -40,34 +38,30 @@ ENV DEBIAN_FRONTEND noninteractive
   
 # nodesource: nvdashboard requires nodejs>=10  
 RUN echo 'deb [trusted=yes] https://developer.download.nvidia.com/hpc-sdk/ubuntu/amd64 /' > /etc/apt/sources.list.d/nvhpc.list
-RUN apt-get update -y && \
-    apt-get install -y -q \
-        apt-utils curl wget
+RUN apt-get update -y && apt-get install -y -q apt-utils curl wget vim
 
 RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
     apt-get install -y -q \
-#        nvhpc-22-5-cuda-multi \
-        nvhpc-22-5 \
-        liblapack-dev  \
-        libblas-dev \
+        liblapack-dev libblas-dev \
         libibverbs-dev libmlx4-1 libmlx5-1 ibutils \
-# Devito Jupyter Notebooks and Ux experience
```

Comment:
Keep the comment

Reply:
Not sure why this comment is here doesn't really gi with what's there imo

Reply:
I would keep the comment about the Ux experience
we should probably also keep those packages because their are for the widget visualisation with dask

Reply:
These packages are still there